grbl/grbl.h

```diff
@@ -22,7 +22,7 @@
 #define grbl_h
 
 // Grbl versioning system
-#define GRBL_VERSION "1.1h-XCP"
+#define GRBL_VERSION "1.1h-XCP-20220314"
```

Comment:
The next `#define` already sets the date. If we're globally including it now then it should be removed. However, lets keep the same format as was previously used [here](https://github.com/inventables/grbl-xcp/blob/564ecb2638e652a15de49f982a186d9cf1c44aef/grbl/report.c#L371) (which also needs to be tweaked).

Reply:
Can you also update the [comment here](https://github.com/inventables/grbl-xcp/blob/master/bootloader/combine.sh#L22)